src/test/java/org/apache/commons/io/input/ReversedLinesFileReaderTestSimple.java

```diff
@@ -66,6 +67,10 @@ public void testUnsupportedEncodingBig5() throws URISyntaxException {
                 () -> new ReversedLinesFileReader(testFileEncodingBig5, IOUtils.DEFAULT_BUFFER_SIZE, "Big5").close());
     }
 
-
+    @Test
+    public void testNullEncoding() throws IOException, URISyntaxException {
+        new ReversedLinesFileReader(new File(this.getClass().getResource("/test-file-empty.bin").toURI()),
+                                    (Charset) null);
+    }
```

Comment:
Hi @mernst,

Thank you for your update to this PR.

The new unit test indeed tests that the ctor does not blow on a null Charset but it does not test that the default Charset kicks in.

Another update would be great.

Reply:
@garydgregory I agree that would be a useful test, even though it is not directly related to this bug fix.

I've made my best guess at how to test for the default Charset in a system-independent way.

I'm not sure whether this is what you had in mind.  If not, could point me at documentation about how Commons IO prefers to test that the default Charset is being used?

Thanks.

Reply:
I agree these tests are useful to improve code coverage.  But they are not testing the change.  Could we create a new pull request or issue to improve the coverage of the test suite, to avoid blocking this pull request?